portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsImageField.testcase

```diff
@@ -131,14 +131,37 @@ definition {
 	}
 
 	@description = "Verify that a Predefined Value can be set"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test PredefinedValueCanBeAdded {
 		property forms.functional = "true";
 		property portal.acceptance = "false";
 
-		// TODO LPS-133049 PredefinedValueCanBeAdded pending implementation
+		ProductMenu.gotoPortlet(
+			category = "Content &amp; Data",
+			portlet = "Forms");
+
+		FormsAdminNavigator.gotoAddForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Image",
+			formPageNumber = "0");
+
+		FormViewBuilder.setFieldPredefinedImage(uploadFileName = "Document_1.jpg");
+
+		Form.save();
+
```

Comment:
@leticiachiu I think step 5 is missing.. `Assert Document_1.jpg is present in the Image field` just after the form is saved.

Reply:
Hello, I think that I did this step inside of macro setImagePredefinedValue
```
AssertElementPresent(
			key_imageName = "${uploadFileName}",
			locator1 = "Form#SIDEBAR_ADVANCED_PREDEFINED_VALUE");
```